### PR TITLE
feat(kubectl-validate): add validate plugin

### PR DESCRIPTION
**Pull request by nlamirault**
Add a new plugin for [kubectl-validate CLI](https://github.com/kubernetes-sigs/kubectl-validate)

Ref: https://github.com/kubernetes-sigs/kubectl-validate/issues/70

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/validate.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: validate
+spec:
+  version: v0.0.3
+  homepage: https://github.com/microsoft/retina
```
Comment (ahmetb) [inline plugins/validate.yaml:7]: seems incorrect

File: plugins/validate.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  shortDescription: a SIG-CLI subproject to support the local validation of resources for native Kubernetes types and CRDs.
```
Comment (ahmetb) [inline plugins/validate.yaml:8]: keep it <50 chars (i.e. cut to the chase)

File: plugins/validate.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  description: |
+    kubectl-validate distinguishes itself among other OpenAPI-based validators by to its deep integration
+    with Kubernetes upstream validation. kubectl-validate is written by Kubernetes apiserver authors
+    using the same code as the server-side.
+    This allows kubectl-validate to give the most accurate error messages and support the Kubernetes-specific
+    validations often ignored by other tools.
```
Comment (ahmetb) [inline plugins/validate.yaml:14]: wrap at 80 chars wide please

File: plugins/validate.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  - selector:
+      matchLabels:
+        os: windows
+        arch: arm64
+    uri: https://github.com/kubernetes-sigs/kubectl-validate/releases/download/v0.0.3/kubectl-validate_windows_arm64.tar.gz
+    sha256: 3a3d660a3bbef8fb4149fbb566fe1899061d27e425815353beb2551b8c20016a
+    bin: kubectl-validate
+
+
+
```
Comment (ahmetb) [inline plugins/validate.yaml:60]: please configure your editor to trim empty newlines. there should be only one \n at the end

File: plugins/validate.yaml (hunk 1 above)
Comment (MPV) [reply plugins/validate.yaml:7]:
```suggestion
  homepage: https://github.com/kubernetes-sigs/kubectl-validate
```

File: plugins/validate.yaml (hunk 4 above)
Comment (MPV) [inline plugins/validate.yaml:60]:
```suggestion

```

**Comment by ahmetb** [comment]
@nlamirault gentle ping. it seems like the build is failing due to a checksum error.

**Comment by MPV** [comment]
Seems like it's green now. 🎉

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**